docs/interpreting-experiments/cuped.md

```diff
@@ -1,15 +1,20 @@
 # Controlled-experiment Using Pre-Existing Data (CUPED)
-CUPED is a statistical tool that leverages pre-existing data to reduce the variance of an A/B experiment. If data is music, CUPED is a pair of noise canceling headphones, allowing you to notice a more pronounced pattern in the data despite a lower volume or strong external noise.
+The precision of experiment results (the width of confidence intervals) depends on the variance in of the metrics we are measuring. One way to improve the precision of experiments is to gather more data, as the variance goes down as we gather more data; this obviously means that it takes longer to run an experiment. 
+
+However, there are options beyond waiting to gather more data, and all these options address reducing the variance in the metrics we are measuring directly. One particular flexible and powerful method is known as CUPED. CUPED is a statistical tool that leverages pre-existing data to reduce the variance of an A/B experiment. You can think of CUPED as a pair of noise cancelling headphones: it uses data gathered prior to the experiment to understand the ambient noise, allowing you to notice a more pronounced pattern in the data.
+
+![CUPED variance reduction explained](../../static/img/measuring-experiments/cuped-variance-plot.png)
+An illustration of how CUPED helps reduce the variance in an effect estimate, leading to a much tighter confidence interval.
 
 Eppo uses CUPED to get more accurate estimates of treatment effect, which should lead to narrower confidence intervals.
 
 ## Overview
-When utilizing CUPED, Eppo automatically ingests event data of subjects in the 30 days prior to experiment start across specified metrics. CUPED works best for experiments with long-time users for whom many pre-experiment data points exist. It is less effective for newer users.
+When utilizing CUPED, Eppo automatically ingests event data of subjects in the 30 days prior to experiment start across metrics that are added to your experiment. Eppo then uses a ridge regression model to predict subject level outcomes for each metric. Relationships between every metric are taken into account to make predictions for other metrics. These predictions are then used to get more accurate effect estimates, reducing the variance and hence leading to better confidence intervals. In particular, our approach allows you to leverage variables with rich datasets to reduce variance for sparser data where pre-experiment datapoints might be lacking (for example, retention and conversion). 
```

Comment:
I think the first sentence is awkward; how about "When you choose to enable CUPED, Eppo will ingest aggregated [? is this true] data about each subject's events (for each metric added to the experiment) for the 30 days before they joined the experiment."

In the next sentence, it should be "subject-level" (hyphen).

For the sentence starting "Relationships", I think it's a bit unclear...how about "For each metric, we predict the outcomes using _every_ available metric."

Then for the last sentence, I think it's confusing that we compare "rich datasets" to "sparser data"...why not use "features" or "metrics"? "In particular, our approach allows you to leverage dense metrics (where subjects have a wide variety of values) to reduce variance for sparser metrics, such as retention and conversion, where only a small fraction of subjects have pre-experiment events.

Reply:
Great stuff; re last comment I think it makes most sense to just remove the last sentence